crates/pop-contracts/src/call.rs

```diff
@@ -130,3 +130,98 @@ pub async fn call_smart_contract(
 		display_events.display_events::<DefaultEnvironment>(Verbosity::Default, &token_metadata)?;
 	Ok(output)
 }
+
+#[cfg(feature = "unit_contract")]
```

Comment:
Feature not defined in crate manifest.

Reply:
True! Well spotted

Reply:
Btw, do we really need this extra feature? If only for the test and not anything else?

Reply:
That's a good point, first approach was to just ignore the very slow "unit tests".
In this discussion https://github.com/r0gue-io/pop-cli/pull/90#pullrequestreview-1975791633 Bruno suggested to use this approach instead of ignoring

Reply:
I understand that. But my concern is now these are libaries. We should select and define `default-features` for these crates &exclude these test-only-features. The end user does not have to know about / run the entire test suite.

Reply:
Are not excluded by default? if they are not indicated in with default = ["unit_contract", "unit_parachain"] ?

Reply:
Yes you're correct. The `default-features` would rather be an indicator that you don't need these additional features.